rust/cubestore/cubestore/src/store/mod.rs

```diff
@@ -780,6 +899,67 @@ impl ChunkStore {
         Ok(new_chunks)
     }
 
+    async fn post_process_columns(
```

Comment:
What does `post_process_columns` function do? Can we please add a small docstring?

Reply:
done at 8cf2c0a